### ISSUE TITLE
Remove `*` resource configuration for role policy

### ISSUE THREAD

**Issue by sbstjn** [sbstjn/serverless-dynamodb-autoscaling#1]
The [permissions for the created role](https://github.com/sbstjn/serverless-dynamodb-autoscaling/blob/master/src/aws/role.js#L31-L44) must have a restricted access policy. The current implementation should not be used in production environments.

```
  'Statement': [
    {
      'Effect': 'Allow',
      'Action': [
        'dynamodb:DescribeTable',
        'dynamodb:UpdateTable',
        'cloudwatch:PutMetricAlarm',
        'cloudwatch:DescribeAlarms',
        'cloudwatch:DeleteAlarms',
        'cloudwatch:GetMetricStatistics',
        'cloudwatch:SetAlarmState'
      ],
      'Resource': '*'
    }
  ]
```

**Comment by sbstjn** [comment]
A first step to a more secure solution would be at least a policy bound to the account and table name:

```
{
            "Action": [
                "dynamodb:DescribeTable",
                "dynamodb:UpdateTable"
            ],
            "Resource": "arn:aws:dynamodb:*:AccountID:table/TableName",
            "Effect": "Allow"
        }
```

**Comment by sbstjn** [comment]
Included in the `0.1.1` release.
